src/styles/index.scss

```diff
@@ -1,3 +1,70 @@
 body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  width: 706px;
+  padding: 0 10px;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__day {
+    align-content: center;
```

Comment:
The 'align-content' property is used here, but it has no effect on flex items. Consider using 'align-items' if you want to align items along the cross axis.